### PR TITLE
Fix #278 and #280, improve setup script, add Py3.6 to AppVeyor

### LINKED ISSUE #278: AttributeError: 'UIAWrapper' object has no attribute 'Menu'
I use backend:uia to print_control_identifiers, and I find the control 'Menu'.
When I trying to do something with the control, I got an error: AttributeError: 'UIAWrapper' object has no attribute 'Menu'.

================================================================
```python
>>> dlg = app.Custom
>>> dlg.print_control_identifiers()
Control Identifiers:

Custom - ''    (L709, T388, R1917, B1033)
['', 'Custom', 'Custom1', '1', 'Custom0', '0']
child_window(auto_id="ControlCenter", control_type="Custom")
   |
   | Button - ''    (L1844, T393, R1864, B413)
   | ['Button0', '2', 'Button1', 'Button']
   | child_window(auto_id="NTWindowButtonMinimize", control_type="Button")
   |
   | Button - ''    (L1868, T393, R1888, B413)
   | ['Button2', '3']
   | child_window(auto_id="NTWindowButtonMaximize", control_type="Button")
   |
   | Button - 'Close'    (L1892, T393, R1912, B413)
   | ['CloseButton', 'Button3', 'Close']
   | child_window(title="Close", auto_id="NTWindowButtonClose", control_type="Button")
   |
   | Menu - ''    (L950, T389, R1680, B417)
   | ['Menu', '4']
   | child_window(auto_id="PART_Menu", control_type="Menu")
   |    |
   |    | MenuItem - 'New'    (L966, T389, R1051, B417)
   |    | ['New0', 'New1', 'New', 'MenuItem0', 'NewMenuItem', 'MenuItem', 'MenuItem1']
   |    | child_window(title="New", auto_id="ControlCenterMenuItemNew", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L974, T393, R998, B413)
   |    |    | ['Static1', '\ue628', '\ue628Static', 'Static0', 'Static']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |    |    |
   |    |    | Static - 'New'    (L1004, T391, R1043, B414)
   |    |    | ['NewStatic', 'Static2', 'New2']
   |    |    | child_window(title="New", control_type="Text")
   |    |
   |    | MenuItem - 'Admin'    (L0, T0, R0, B0)
   |    | ['Admin', 'MenuItem2', 'AdminMenuItem']
   |    | child_window(title="Admin", auto_id="ControlCenterMenuItemAdmin", control_type="MenuItem")
   |    |
   |    | MenuItem - 'Tools'    (L1066, T389, R1164, B417)
   |    | ['ToolsMenuItem', 'Tools', 'Tools1', 'Tools0', 'MenuItem3']
   |    | child_window(title="Tools", auto_id="ControlCenterMenuItemTools", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L1074, T393, R1094, B413)
   |    |    | ['Static3', '\ue01e', '\ue01eStatic']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |    |    |
   |    |    | Static - 'Tools'    (L1104, T391, R1156, B414)
   |    |    | ['ToolsStatic', 'Tools2', 'Static4']
   |    |    | child_window(title="Tools", control_type="Text")
   |    |
   |    | MenuItem - 'Workspaces'    (L1178, T389, R1339, B417)
   |    | ['MenuItem4', 'Workspaces0', 'Workspaces1', 'Workspaces', 'WorkspacesMenuItem']
   |    | child_window(title="Workspaces", auto_id="ControlCenterMenuItemWorkspaces", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L1186, T393, R1206, B413)
   |    |    | ['\ue01fStatic', '\ue01f', 'Static5']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |    |    |
   |    |    | Static - 'Workspaces'    (L1216, T391, R1331, B414)
   |    |    | ['Static6', 'Workspaces2', 'WorkspacesStatic']
   |    |    | child_window(title="Workspaces", control_type="Text")
   |    |
   |    | MenuItem - 'Connections'    (L1354, T389, R1518, B417)
   |    | ['Connections1', 'MenuItem5', 'Connections0', 'ConnectionsMenuItem', 'Connections']
   |    | child_window(title="Connections", auto_id="ControlCenterMenuItemConnections", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L1362, T393, R1382, B413)
   |    |    | ['Static7', '\ue020', '\ue020Static']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |    |    |
   |    |    | Static - 'Connections'    (L1392, T391, R1510, B414)
   |    |    | ['ConnectionsStatic', 'Static8', 'Connections2']
   |    |    | child_window(title="Connections", control_type="Text")
   |    |
   |    | MenuItem - 'Help'    (L1534, T389, R1621, B417)
   |    | ['HelpMenuItem0', 'Help1', 'Help0', 'Help', 'HelpMenuItem1', 'HelpMenuItem', 'MenuItem6']
   |    | child_window(title="Help", auto_id="ControlCenterMenuItemHelp", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L1542, T393, R1562, B413)
   |    |    | ['\ue021Static', '\ue021', 'Static9']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |    |    |
   |    |    | Static - 'Help'    (L1572, T391, R1613, B414)
   |    |    | ['Help2', 'Static10', 'HelpStatic']
   |    |    | child_window(title="Help", control_type="Text")
   |    |
   |    | MenuItem - ''    (L1636, T393, R1674, B413)
   |    | ['5', 'MenuItem7', 'HelpMenuItem2']
   |    | child_window(auto_id="ControlCenterMenuItemLinks", control_type="MenuItem")
   |    |    |
   |    |    | Static - ''    (L1644, T393, R1656, B413)
   |    |    | ['\ue800Static', 'Static11', '\ue800']
   |    |    | child_window(title="", auto_id="txtBlockIconAsFontTemplate", control_type="Text")
   |
   | Toolbar - ''    (L0, T0, R0, B0)
   | ['Toolbar', '6']
   | child_window(auto_id="Control CenterWindowToolBar", control_type="ToolBar")
   |
   | TabControl - ''    (L718, T426, R1909, B1025)
   | ['TabControlExecutions', 'TabControl', 'TabControlOrders', 'TabControlLog', 'TabControlPositions', 'TabControlSt
egies', '7']
   | child_window(auto_id="ControlCenterTabControl", control_type="Tab")
   |    |
   |    | TabItem - 'Accounts'    (L769, T984, R866, B1025)
   |    | ['AccountsTabItem', 'Accounts0', 'TabItem0', 'Accounts1', 'TabItem', 'TabItem1', 'Accounts']
   |    | child_window(title="Accounts", auto_id="AccountsGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Accounts'    (L777, T992, R859, B1014)
   |    |    | ['AccountsStatic0', 'Static12', 'AccountsStatic1', 'AccountsStatic', 'Accounts2']
   |    |    | child_window(title="Accounts", control_type="Text")
   |    |    |
   |    |    | Static - 'Accounts'    (L777, T994, R841, B1013)
   |    |    | ['Static13', 'AccountsStatic2', 'Accounts3']
   |    |    | child_window(title="Accounts", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L848, T996, R863, B1011)
   |    |    | ['8', 'Button4', 'AccountsButton']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")
   |    |
   |    | TabItem - 'Strategies'    (L867, T984, R971, B1025)
   |    | ['TabItem2', 'StrategiesTabItem', 'Strategies', 'Strategies1', 'Strategies0']
   |    | child_window(title="Strategies", auto_id="StrategiesGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Strategies'    (L875, T992, R964, B1014)
   |    |    | ['Static14', 'StrategiesStatic', 'Strategies2', 'StrategiesStatic0', 'StrategiesStatic1']
   |    |    | child_window(title="Strategies", control_type="Text")
   |    |    |
   |    |    | Static - 'Strategies'    (L875, T994, R947, B1013)
   |    |    | ['StrategiesStatic2', 'Strategies3', 'Static15']
   |    |    | child_window(title="Strategies", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L953, T996, R968, B1011)
   |    |    | ['9', 'StrategiesButton', 'Button5']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")
   |    |
   |    | TabItem - 'Orders'    (L972, T984, R1062, B1025)
   |    | ['TabItem3', 'OrdersTabItem', 'Orders', 'Orders1', 'Orders0']
   |    | child_window(title="Orders", auto_id="OrdersGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Orders'    (L980, T992, R1040, B1014)
   |    |    | ['OrdersStatic', 'OrdersStatic0', 'Orders2', 'Static16', 'OrdersStatic1']
   |    |    | child_window(title="Orders", control_type="Text")
   |    |    |
   |    |    | Static - 'Orders'    (L980, T994, R1032, B1013)
   |    |    | ['Static17', 'Orders3', 'OrdersStatic2']
   |    |    | child_window(title="Orders", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L1044, T996, R1059, B1011)
   |    |    | ['Button6', '10', 'OrdersButton']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")
   |    |    |
   |    |    | Custom - ''    (L719, T427, R1908, B984)
   |    |    | ['11', 'Custom2']
   |    |    | child_window(auto_id="OrdersGrid", control_type="Custom")
   |    |    |    |
   |    |    |    | Custom - ''    (L719, T427, R1908, B984)
   |    |    |    | ['Custom3', '12']
   |    |    |    | child_window(auto_id="OrderGridTable", control_type="Custom")
   |    |    |    |    |
   |    |    |    |    | ListView - 'Records'    (L719, T427, R1909, B984)
   |    |    |    |    | ['RecordsListView', 'Records', 'ListView']
   |    |    |    |    | child_window(title="Records", control_type="DataGrid")
   |    |    |    |    |    |
   |    |    |    |    |    | Header - 'MainFieldLayout - 23 Fields - index = 0'    (L719, T427, R1909, B460)
   |    |    |    |    |    | ['MainFieldLayout - 23 Fields - index = 0Header', 'MainFieldLayout - 23 Fields - index
', 'Header']
   |    |    |    |    |    | child_window(title="MainFieldLayout - 23 Fields - index = 0", control_type="Header")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | Thumb - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    | ['Thumb0', 'Thumb1', '13', 'Thumb']
   |    |    |    |    |    |    | child_window(auto_id="Near", control_type="Thumb")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | Thumb - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    | ['Thumb2', '14']
   |    |    |    |    |    |    | child_window(auto_id="Far", control_type="Thumb")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Instrument'    (L719, T427, R781, B459)
   |    |    |    |    |    |    | ['Instrument', 'HeaderItem0', 'InstrumentHeaderItem', 'HeaderItem', 'HeaderItem1',
nstrument0', 'Instrument1']
   |    |    |    |    |    |    | child_window(title="Instrument", auto_id="HeaderRowInstrument", control_type="Head
tem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Instrument'    (L727, T435, R796, B452)
   |    |    |    |    |    |    |    | ['InstrumentStatic', 'Static18', 'Instrument2']
   |    |    |    |    |    |    |    | child_window(title="Instrument", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['15', 'Button7']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Action'    (L781, T427, R844, B459)
   |    |    |    |    |    |    | ['Action0', 'Action1', 'Action', 'HeaderItem2', 'ActionHeaderItem']
   |    |    |    |    |    |    | child_window(title="Action", auto_id="HeaderRowOrderAction", control_type="HeaderI
")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Action'    (L789, T435, R829, B452)
   |    |    |    |    |    |    |    | ['ActionStatic', 'Static19', 'Action2']
   |    |    |    |    |    |    |    | child_window(title="Action", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button8', '16']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Type'    (L844, T427, R906, B459)
   |    |    |    |    |    |    | ['HeaderItem3', 'Type1', 'Type0', 'Type', 'TypeHeaderItem']
   |    |    |    |    |    |    | child_window(title="Type", auto_id="HeaderRowOrderType", control_type="HeaderItem"
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Type'    (L851, T435, R882, B452)
   |    |    |    |    |    |    |    | ['Static20', 'TypeStatic', 'Type2']
   |    |    |    |    |    |    |    | child_window(title="Type", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L890, T441, R899, B445)
   |    |    |    |    |    |    |    | ['TypeButton', '17', 'Button9']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Quantity'    (L906, T427, R969, B459)
   |    |    |    |    |    |    | ['QuantityHeaderItem', 'HeaderItem4', 'Quantity1', 'Quantity', 'Quantity0']
   |    |    |    |    |    |    | child_window(title="Quantity", auto_id="HeaderRowTotalQuantity", control_type="Hea
Item")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Quantity'    (L914, T435, R968, B452)
   |    |    |    |    |    |    |    | ['QuantityStatic', 'Static21', 'Quantity2']
   |    |    |    |    |    |    |    | child_window(title="Quantity", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button10', '18']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Limit'    (L969, T427, R1031, B459)
   |    |    |    |    |    |    | ['Limit', 'HeaderItem5', 'Limit0', 'Limit1', 'LimitHeaderItem']
   |    |    |    |    |    |    | child_window(title="Limit", auto_id="HeaderRowLimitPrice", control_type="HeaderIte
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Limit'    (L976, T435, R1007, B452)
   |    |    |    |    |    |    |    | ['LimitStatic', 'Limit2', 'Static22']
   |    |    |    |    |    |    |    | child_window(title="Limit", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button11', '19']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Stop'    (L1031, T427, R1094, B459)
   |    |    |    |    |    |    | ['Stop1', 'HeaderItem6', 'Stop0', 'StopHeaderItem', 'Stop']
   |    |    |    |    |    |    | child_window(title="Stop", auto_id="HeaderRowStopPrice", control_type="HeaderItem"
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Stop'    (L1039, T435, R1069, B452)
   |    |    |    |    |    |    |    | ['Stop2', 'Static23', 'StopStatic']
   |    |    |    |    |    |    |    | child_window(title="Stop", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button12', '20']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'State'    (L1094, T427, R1222, B459)
   |    |    |    |    |    |    | ['HeaderItem7', 'State0', 'State1', 'StateHeaderItem', 'State']
   |    |    |    |    |    |    | child_window(title="State", auto_id="HeaderRowOrderStateString", control_type="Hea
Item")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'State'    (L1101, T435, R1135, B452)
   |    |    |    |    |    |    |    | ['State2', 'StateStatic', 'Static24']
   |    |    |    |    |    |    |    | child_window(title="State", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button13', '21']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Filled'    (L1222, T427, R1284, B459)
   |    |    |    |    |    |    | ['FilledHeaderItem', 'Filled', 'HeaderItem8', 'Filled0', 'Filled1']
   |    |    |    |    |    |    | child_window(title="Filled", auto_id="HeaderRowFilled", control_type="HeaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Filled'    (L1229, T435, R1263, B452)
   |    |    |    |    |    |    |    | ['Static25', 'FilledStatic', 'Filled2']
   |    |    |    |    |    |    |    | child_window(title="Filled", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['22', 'Button14']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Avg. price'    (L1284, T427, R1347, B459)
   |    |    |    |    |    |    | ['Avg. price0', 'Avg. price', 'Avg. price1', 'Avg. priceHeaderItem', 'HeaderItem9'
   |    |    |    |    |    |    | child_window(title="Avg. price", auto_id="HeaderRowAverageFillPrice", control_type
eaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Avg. price'    (L1292, T435, R1356, B452)
   |    |    |    |    |    |    |    | ['Avg. price2', 'Static26', 'Avg. priceStatic']
   |    |    |    |    |    |    |    | child_window(title="Avg. price", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button15', '23']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Remaining'    (L1347, T427, R1409, B459)
   |    |    |    |    |    |    | ['HeaderItem10', 'Remaining', 'Remaining1', 'RemainingHeaderItem', 'Remaining0']
   |    |    |    |    |    |    | child_window(title="Remaining", auto_id="HeaderRowRemaining", control_type="Header
m")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Remaining'    (L1354, T435, R1424, B452)
   |    |    |    |    |    |    |    | ['RemainingStatic', 'Static27', 'Remaining2']
   |    |    |    |    |    |    |    | child_window(title="Remaining", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['24', 'Button16']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Name'    (L1409, T427, R1472, B459)
   |    |    |    |    |    |    | ['Name0', 'Name', 'Name1', 'HeaderItem11', 'NameHeaderItem']
   |    |    |    |    |    |    | child_window(title="Name", auto_id="HeaderRowOrderName", control_type="HeaderItem"
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Name'    (L1417, T435, R1456, B452)
   |    |    |    |    |    |    |    | ['Name2', 'NameStatic', 'Static28']
   |    |    |    |    |    |    |    | child_window(title="Name", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button17', '25']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Strategy'    (L1472, T427, R1534, B459)
   |    |    |    |    |    |    | ['StrategyHeaderItem', 'Strategy', 'HeaderItem12', 'Strategy0', 'Strategy1']
   |    |    |    |    |    |    | child_window(title="Strategy", auto_id="HeaderRowStrategyString", control_type="He
rItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Strategy'    (L1479, T435, R1533, B452)
   |    |    |    |    |    |    |    | ['Strategy2', 'StrategyStatic', 'Static29']
   |    |    |    |    |    |    |    | child_window(title="Strategy", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['26', 'Button18']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'OCO'    (L1534, T427, R1597, B459)
   |    |    |    |    |    |    | ['OCO0', 'OCO', 'OCOHeaderItem', 'OCO1', 'HeaderItem13']
   |    |    |    |    |    |    | child_window(title="OCO", auto_id="HeaderRowOneCancelsOther", control_type="Header
m")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'OCO'    (L1542, T435, R1577, B452)
   |    |    |    |    |    |    |    | ['OCO2', 'OCOStatic', 'Static30']
   |    |    |    |    |    |    |    | child_window(title="OCO", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button19', '27']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'TIF'    (L1597, T427, R1659, B459)
   |    |    |    |    |    |    | ['TIF1', 'TIF', 'HeaderItem14', 'TIFHeaderItem', 'TIF0']
   |    |    |    |    |    |    | child_window(title="TIF", auto_id="HeaderRowTimeInForce", control_type="HeaderItem
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'TIF'    (L1604, T435, R1625, B452)
   |    |    |    |    |    |    |    | ['TIFStatic', 'Static31', 'TIF2']
   |    |    |    |    |    |    |    | child_window(title="TIF", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['28', 'Button20']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Account display name'    (L1659, T427, R1722, B459)
   |    |    |    |    |    |    | ['Account display name0', 'Account display name', 'Account display nameHeaderItem'
Account display name1', 'HeaderItem15']
   |    |    |    |    |    |    | child_window(title="Account display name", auto_id="HeaderRowAccount", control_typ
HeaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Account display name'    (L1667, T435, R1810, B452)
   |    |    |    |    |    |    |    | ['Account display nameStatic', 'Static32', 'Account display name2']
   |    |    |    |    |    |    |    | child_window(title="Account display name", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['29', 'Button21']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'ID'    (L1722, T427, R1784, B459)
   |    |    |    |    |    |    | ['HeaderItem16', 'ID0', 'ID1', 'ID', 'IDHeaderItem']
   |    |    |    |    |    |    | child_window(title="ID", auto_id="HeaderRowOrderId", control_type="HeaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'ID'    (L1729, T435, R1743, B452)
   |    |    |    |    |    |    |    | ['ID2', 'IDStatic', 'Static33']
   |    |    |    |    |    |    |    | child_window(title="ID", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['30', 'Button22']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Time'    (L1784, T427, R1847, B459)
   |    |    |    |    |    |    | ['HeaderItem17', 'TimeHeaderItem', 'Time1', 'Time', 'Time0']
   |    |    |    |    |    |    | child_window(title="Time", auto_id="HeaderRowTime", control_type="HeaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Time'    (L1792, T435, R1824, B452)
   |    |    |    |    |    |    |    | ['TimeStatic', 'Time2', 'Static34']
   |    |    |    |    |    |    |    | child_window(title="Time", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button23', '31']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |    |    |    |    |    |
   |    |    |    |    |    |    | HeaderItem - 'Cancel'    (L1847, T427, R1909, B459)
   |    |    |    |    |    |    | ['CancelHeaderItem', 'Cancel1', 'Cancel', 'Cancel0', 'HeaderItem18']
   |    |    |    |    |    |    | child_window(title="Cancel", auto_id="HeaderRowX", control_type="HeaderItem")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Static - 'Cancel'    (L1854, T435, R1900, B452)
   |    |    |    |    |    |    |    | ['Static35', 'CancelStatic', 'Cancel2']
   |    |    |    |    |    |    |    | child_window(title="Cancel", control_type="Text")
   |    |    |    |    |    |    |    |
   |    |    |    |    |    |    |    | Button - ''    (L0, T0, R0, B0)
   |    |    |    |    |    |    |    | ['Button24', '32']
   |    |    |    |    |    |    |    | child_window(auto_id="SortIndicator", control_type="Button")
   |    |
   |    | TabItem - 'Executions'    (L1063, T984, R1176, B1025)
   |    | ['ExecutionsTabItem', 'Executions', 'TabItem4', 'Executions0', 'Executions1']
   |    | child_window(title="Executions", auto_id="ExecutionsGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Executions'    (L1071, T992, R1169, B1014)
   |    |    | ['ExecutionsStatic0', 'ExecutionsStatic', 'ExecutionsStatic1', 'Static36', 'Executions2']
   |    |    | child_window(title="Executions", control_type="Text")
   |    |    |
   |    |    | Static - 'Executions'    (L1071, T994, R1147, B1013)
   |    |    | ['Static37', 'ExecutionsStatic2', 'Executions3']
   |    |    | child_window(title="Executions", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L1158, T996, R1173, B1011)
   |    |    | ['ExecutionsButton', '33', 'Button25']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")
   |    |
   |    | TabItem - 'Positions'    (L1177, T984, R1273, B1025)
   |    | ['PositionsTabItem', 'Positions0', 'Positions1', 'TabItem5', 'Positions']
   |    | child_window(title="Positions", auto_id="PositionsGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Positions'    (L1185, T992, R1266, B1014)
   |    |    | ['PositionsStatic0', 'Positions2', 'PositionsStatic', 'PositionsStatic1', 'Static38']
   |    |    | child_window(title="Positions", control_type="Text")
   |    |    |
   |    |    | Static - 'Positions'    (L1185, T994, R1250, B1013)
   |    |    | ['Static39', 'Positions3', 'PositionsStatic2']
   |    |    | child_window(title="Positions", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L1255, T996, R1270, B1011)
   |    |    | ['PositionsButton', 'Button26', '34']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")
   |    |
   |    | TabItem - 'Log'    (L1274, T984, R1364, B1025)
   |    | ['Log1', 'LogTabItem', 'Log0', 'Log', 'TabItem6']
   |    | child_window(title="Log", auto_id="LogGridTabItem", control_type="TabItem")
   |    |    |
   |    |    | Static - 'Log'    (L1282, T992, R1315, B1014)
   |    |    | ['LogStatic', 'LogStatic0', 'LogStatic1', 'Log2', 'Static40']
   |    |    | child_window(title="Log", control_type="Text")
   |    |    |
   |    |    | Static - 'Log'    (L1282, T994, R1309, B1013)
   |    |    | ['LogStatic2', 'Log3', 'Static41']
   |    |    | child_window(title="Log", control_type="Text")
   |    |    |
   |    |    | Button - ''    (L1346, T996, R1361, B1011)
   |    |    | ['35', 'LogButton', 'Button27']
   |    |    | child_window(auto_id="TabControlButtonCloseTab", control_type="Button")

>>> dlg = app.Custom.Menu
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files\Python35\lib\site-packages\pywinauto\application.py", line 359, in __getattribute__
    return getattr(ctrls[-1], attr_name)
AttributeError: 'UIAWrapper' object has no attribute 'Menu'
```

### PR DESCRIPTION
**Pull request by vasily-v-ryabov**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -61,6 +61,16 @@ environment:
...
+    - PYTHON: "C:\\Python36"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "32"
+      UIA_SUPPORT: "NO"
+
+    - PYTHON: "C:\\Python36-x64"
+      PYTHON_VERSION: "3.6"
+      PYTHON_ARCH: "64"
+      UIA_SUPPORT: "YES"
+
```
Comment (airelil) [inline appveyor.yml]: What about dropping out the Python 3.3 from the matrix ?

File: pywinauto/application.py [hunk 2]
```diff
@@ -356,10 +356,14 @@ def __getattribute__(self, attr_name):
         else:
             # if we have been asked for an attribute of the dialog
             # then resolve the window and return the attribute
-            if len(self.criteria) == 1 and hasattr(DialogWrapper, attr_name):
+            desktop_wrapper = self.backend.generic_wrapper_class(self.backend.element_info_class())
+            need_to_resolve = (len(self.criteria) == 1 and hasattr(desktop_wrapper, attr_name))
+            if self.backend.name == 'win32':
```
Comment (airelil) [inline pywinauto/application.py]: Probably ```DialogWrapper``` import should be moved here too. Check a line above:
```from .controls.win32_controls import DialogWrapper```

File: setup.py [hunk 3]
```diff
@@ -93,12 +93,12 @@ def setup_path(path = ""):
         'Programming Language :: Python :: 3.3',
```
Comment (airelil) [inline setup.py]: Again, time to drop it ?

File: dev-requirements.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
 pypiwin32
 six
-Pillow==3.0.0
+Pillow==4.0.0
```
Comment (airelil) [inline dev-requirements.txt]: Is it must 4.0 ? Could it be just Pillow >= 3.0 ?

File: appveyor.yml (hunk 1 above)
Comment (vasily-v-ryabov) [inline appveyor.yml]: Not sure it's time to drop Py3.3 support. Maybe later. And... more pythons - more chances to reproduce unstable test cases that we fight with.

File: pywinauto/application.py (hunk 2 above)
Comment (vasily-v-ryabov) [inline pywinauto/application.py]: Good point. I will try to make even better fix that will fully move this import from `application.py`.

File: dev-requirements.txt (hunk 4 above)
Comment (vasily-v-ryabov) [inline dev-requirements.txt]: It's for Py3.6 compatibility. Doesn't work with Pillow==3.0.0 or even 3.4.2 (no wheels for py36).
Pillow==4.0.0 contains wheels for all pythons: 2.7 and 3.3+.

**Comment by vasily-v-ryabov** [comment]
Travis CI tests can run now but have one unstable test. Had to leave this as is. It's not a high priority for now.

**Pull request merged**